tools/unleash-server-utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const execa = require('execa');
+
+/**
+ * Gets the installed version of unleash-server.
+ *
+ * @returns {Promise<string>}
+ */
+async function getUnleashServerVersion() {
+  const { stdout: npmListOutput } = await execa.command(
+    'npm list unleash-server',
+  );
+  const versionLine = npmListOutput.split('\n')[1];
+  return versionLine.split('@')[1].trim();
+}
```

Comment:
this is to include the unleash-server version in the Docker tag. It's not pretty, but I don't know a better way to find the installed version. Do you?

(the `.trim()` gets rid of a trailing space)

Reply:
`require('unleash-server/package.json').version`

Reply:
yep that's approximately 78 times better. b6e864cf0e5acc9bcd422d91b063d4ef1fbc7ebc